### ISSUE TITLE
Error Restoring Backup in New Server

### ISSUE THREAD

**Issue by pwrj** [AzuraCast/AzuraCast#4572]
### Installation Method

Docker Installation

### AzuraCast Release Channel

Stable Channel

### Current AzuraCast Version

v0.14.1

### What happened?

Error during restore from backup. Help please... Backup with media (23 GB)

### Relevant log output

```shell
In AbstractMySQLDriver.php line 79:

  An exception occurred while executing 'SELECT t0.app_unique_identifier AS app_unique_identifier_1, t0.base_url AS base_url_2, t0.instance_name AS instance_name_3, t0.prefer_browser_ur
  l AS prefer_browser_url_4, t0.use_radio_proxy AS use_radio_proxy_5, t0.history_keep_days AS history_keep_days_6, t0.always_use_ssl AS always_use_ssl_7, t0.api_access_control AS api_ac
  cess_control_8, t0.enable_websockets AS enable_websockets_9, t0.analytics AS analytics_10, t0.check_for_updates AS check_for_updates_11, t0.update_results AS update_results_12, t0.upd
  ate_last_run AS update_last_run_13, t0.public_theme AS public_theme_14, t0.hide_album_art AS hide_album_art_15, t0.homepage_redirect_url AS homepage_redirect_url_16, t0.default_album_
  art_url AS default_album_art_url_17, t0.use_external_album_art_when_processing_media AS use_external_album_art_when_processing_media_18, t0.use_external_album_art_in_apis AS use_exter
  nal_album_art_in_apis_19, t0.last_fm_api_key AS last_fm_api_key_20, t0.hide_product_name AS hide_product_name_21, t0.public_custom_css AS public_custom_css_22, t0.public_custom_js AS
  public_custom_js_23, t0.internal_custom_css AS internal_custom_css_24, t0.backup_enabled AS backup_enabled_25, t0.backup_time_code AS backup_time_code_26, t0.backup_exclude_media AS b
  ackup_exclude_media_27, t0.backup_keep_copies AS backup_keep_copies_28, t0.backup_storage_location AS backup_storage_location_29, t0.backup_last_run AS backup_last_run_30, t0.backup_l
  ast_result AS backup_last_result_31, t0.backup_last_output AS backup_last_output_32, t0.setup_complete_time AS setup_complete_time_33, t0.nowplaying AS nowplaying_34, t0.sync_nowplayi
  ng_last_run AS sync_nowplaying_last_run_35, t0.sync_short_last_run AS sync_short_last_run_36, t0.sync_medium_last_run AS sync_medium_last_run_37, t0.sync_long_last_run AS sync_long_la
  st_run_38, t0.external_ip AS external_ip_39, t0.geolite_license_key AS geolite_license_key_40, t0.geolite_last_run AS geolite_last_run_41, t0.enable_advanced_features AS enable_advanc
  ed_features_42, t0.mail_enabled AS mail_enabled_43, t0.mail_sender_name AS mail_sender_name_44, t0.mail_sender_email AS mail_sender_email_45, t0.mail_smtp_host AS mail_smtp_host_46, t
  0.mail_smtp_port AS mail_smtp_port_47, t0.mail_smtp_username AS mail_smtp_username_48, t0.mail_smtp_password AS mail_smtp_password_49, t0.mail_smtp_secure AS mail_smtp_secure_50, t0.a
  vatar_service AS avatar_service_51, t0.avatar_default_url AS avatar_default_url_52 FROM settings t0 LIMIT 1':

  SQLSTATE[42S22]: Column not found: 1054 Unknown column 't0.backup_last_result' in 'field list'


In Exception.php line 18:

  SQLSTATE[42S22]: Column not found: 1054 Unknown column 't0.backup_last_result' in 'field list'


In PDOConnection.php line 132:

  SQLSTATE[42S22]: Column not found: 1054 Unknown column 't0.backup_last_result' in 'field list'
```

**Comment by SC2Mitch** [comment]
This kind of issue indicates the installation process didn't fully work, can you re-run a update to see if this resolves it?

**Comment by pwrj** [comment]
@SC2Mitch I ran update. Same...

In AbstractMySQLDriver.php line 79:

  An exception occurred while executing 'SELECT t0.app_unique_identifier AS app_unique_identifier_1, t0.base_url AS base_url_2, t0.instance_name AS instance_name_3, t0.prefer_browser_ur
  l AS prefer_browser_url_4, t0.use_radio_proxy AS use_radio_proxy_5, t0.history_keep_days AS history_keep_days_6, t0.always_use_ssl AS always_use_ssl_7, t0.api_access_control AS api_ac
  cess_control_8, t0.enable_websockets AS enable_websockets_9, t0.analytics AS analytics_10, t0.check_for_updates AS check_for_updates_11, t0.update_results AS update_results_12, t0.upd
  ate_last_run AS update_last_run_13, t0.public_theme AS public_theme_14, t0.hide_album_art AS hide_album_art_15, t0.homepage_redirect_url AS homepage_redirect_url_16, t0.default_album_
  art_url AS default_album_art_url_17, t0.use_external_album_art_when_processing_media AS use_external_album_art_when_processing_media_18, t0.use_external_album_art_in_apis AS use_exter
  nal_album_art_in_apis_19, t0.last_fm_api_key AS last_fm_api_key_20, t0.hide_product_name AS hide_product_name_21, t0.public_custom_css AS public_custom_css_22, t0.public_custom_js AS
  public_custom_js_23, t0.internal_custom_css AS internal_custom_css_24, t0.backup_enabled AS backup_enabled_25, t0.backup_time_code AS backup_time_code_26, t0.backup_exclude_media AS b
  ackup_exclude_media_27, t0.backup_keep_copies AS backup_keep_copies_28, t0.backup_storage_location AS backup_storage_location_29, t0.backup_last_run AS backup_last_run_30, t0.backup_l
  ast_result AS backup_last_result_31, t0.backup_last_output AS backup_last_output_32, t0.setup_complete_time AS setup_complete_time_33, t0.nowplaying AS nowplaying_34, t0.sync_nowplayi
  ng_last_run AS sync_nowplaying_last_run_35, t0.sync_short_last_run AS sync_short_last_run_36, t0.sync_medium_last_run AS sync_medium_last_run_37, t0.sync_long_last_run AS sync_long_la
  st_run_38, t0.external_ip AS external_ip_39, t0.geolite_license_key AS geolite_license_key_40, t0.geolite_last_run AS geolite_last_run_41, t0.enable_advanced_features AS enable_advanc
  ed_features_42, t0.mail_enabled AS mail_enabled_43, t0.mail_sender_name AS mail_sender_name_44, t0.mail_sender_email AS mail_sender_email_45, t0.mail_smtp_host AS mail_smtp_host_46, t
  0.mail_smtp_port AS mail_smtp_port_47, t0.mail_smtp_username AS mail_smtp_username_48, t0.mail_smtp_password AS mail_smtp_password_49, t0.mail_smtp_secure AS mail_smtp_secure_50, t0.a
  vatar_service AS avatar_service_51, t0.avatar_default_url AS avatar_default_url_52 FROM settings t0 LIMIT 1':

  SQLSTATE[42S22]: Column not found: 1054 Unknown column 't0.backup_last_result' in 'field list'


In Exception.php line 18:

  SQLSTATE[42S22]: Column not found: 1054 Unknown column 't0.backup_last_result' in 'field list'


In PDOConnection.php line 132:

  SQLSTATE[42S22]: Column not found: 1054 Unknown column 't0.backup_last_result' in 'field list'


azuracast:setup [--update] [--load-fixtures] [--release]

**Comment by Vaalyn** [comment]
Are you using the steps outlined [here](https://docs.azuracast.com/en/administration/backup-and-restore#restoring-a-backup) in our docs?

Is the backup file from the same version AzuraCast that you are trying to restore it to?
Missing DB columns/tables can happen if the version of the backup is different from the one on the server you want to restore it to. I've seen a few cases where this happened so I modified our restore instructions some time back.

**Comment by pwrj** [comment]
@Vaalyn You might be right. Let me try again.

**Comment by pwrj** [comment]
@Vaalyn You were absolutely correct my friend. I made sure the backup is the same version as the new azuracast install and boom! It went without a hitch.  Thank you @Vaalyn @SC2Mitch 👍🏾

**Comment by Vaalyn** [comment]
> @Vaalyn You were absolutely correct my friend. I made sure the backup is the same version as the new azuracast install and boom! It went without a hitch. Thank you @Vaalyn @SC2Mitch 👍🏾

Great to hear it worked 👍
